docs/docs/examples/olymp-file-sharing.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: 'Basic: Olymp Library'
+---
+
+import RelationTuplePrism from '@theme/RelationTuplePrism'
+RelationTuplePrism()
+
+#### A basic, down-to-earth full feature example
+
+Consider a file sharing application called "Olymp Library". Each file is stored
+in a key-value store, where the key is a UUIDv4 (pseudorandom unique
+identifier), while the value is the metadata and content. The application uses
+Ory Keto to keep track of ownership and granted access on a per file level.
+
+:::note
+
+This example assumes there is a [namespace](../concepts/namespace) `files` with
```

Comment:
For these links to properly work, you need to append the filename. That helps docusaurus properly resolve the filename. Please do this for all links! You can use search with regexp.